src/utils/sources-tree/utils.js

```diff
@@ -51,7 +50,9 @@ export function isSource(item: TreeNode) {
 }
 
 export function getFileExtension(source: Source): string {
-  const parsedUrl = getURL(source).path;
+  const strippedUrl = source.url.replace(/ \[sm\]/, "");
+  const parsedUrl = parse(strippedUrl).pathname;
```

Comment:
maybe we should do this inside of `parse(source)`, so that every source has the `[sm]` stripped out of the fields?